bitstreams.md

```diff
@@ -37,14 +37,87 @@ Exposed links:
 
 Bitstream metadata can be modified as described in [Modifying metadata via Patch](metadata-patch.md).
 
+Additional bitstream properties can be modified via Patch as described below.
+
+### Replace
+The replace operation allows to replace *existent* information with new one. Attempt to use the replace operation to set not yet initialized information must return an error. See [general errors on PATCH requests](patch.md)
+
+To change the bundle of a bitstream:
+
+`curl --data '[ { "op": "replace", "path": "/bundleName", "value": "TEST"}]' -H "Authorization: Bearer ..." -H "content-type: application/json" -X PATCH ${dspace7-url}/api/core/bitstreams/${uuid}`
```

Comment:
due to the agreement of the Bundle DS-4317 should we withdrawn this operation from the contract?

Reply:
I've removed this feature, there's no use in implementing it and removing it again for https://jira.duraspace.org/browse/DS-4317